src/lib/InPort.coffee

```diff
@@ -32,6 +32,7 @@ class InPort extends BasePort
       @process = process
 
     if options.handle
+      platform.deprecated 'InPort handle callback is deprecated. Please use Process API'
```

Comment:
I consider it somewhat dangerous, given that it doesn't actually read from the port buffer, and doesn't provide any activation/deactivation logic.

If users must, they can of course still do `component.inPorts.foo.on 'ip', (ip) -> # do custom stuff`